graphjet-core/src/main/java/com/twitter/graphjet/algorithms/RecommendationRequest.java

```diff
@@ -26,6 +26,7 @@
   private final long queryNode;
   private final LongSet toBeFiltered;
   private final byte[] socialProofTypes;
+  public static final int MAX_RECOMMENDATION_RESULTS = 1000;
```

Comment:
in addition to MAX_RECOMMENDATION_RESULTS, could we have DEFAULT_RECOMMENDATION_RESULTS set to 100 so that we don't have to return this many results all the time?